packages/dd-trace/src/appsec/recommended.json

```diff
@@ -0,0 +1 @@
xx","name":"CGIchk","tags":{"type":"security_scanner","category":"attack_attempt"},"conditions":[{"parameters":{"inputs":[{"address":"server.request.headers.no_cookies","key_path":["user-agent"]}],"regex":"(?i)\\bcgichk\\b"},"operator":"match_regex"}],"transformers":[]},{"id":"ua0-600-51x","name":"FFUF","tags":{"type":"security_scanner","category":"attack_attempt"},"conditions":[{"parameters":{"inputs":[{"address":"server.request.headers.no_cookies","key_path":["user-agent"]}],"regex":"(?i)^Fuzz Faster U Fool\\b"},"operator":"match_regex"}],"transformers":[]},{"id":"ua0-600-52x","name":"Nuclei is a fast tool for configurable targeted scanning based on templates offering massive extensibility and ease of use","tags":{"type":"security_scanner","category":"attack_attempt"},"conditions":[{"parameters":{"inputs":[{"address":"server.request.headers.no_cookies","key_path":["user-agent"]}],"regex":"(?i)^Nuclei\\b"},"operator":"match_regex"}],"transformers":[]},{"id":"ua0-600-53x","name":"Tsunami is a general purpose network security scanner with an extensible plugin system for detecting high severity vulnerabilities with high confidence","tags":{"type":"security_scanner","category":"attack_attempt"},"conditions":[{"parameters":{"inputs":[{"address":"server.request.headers.no_cookies","key_path":["user-agent"]}],"regex":"(?i)\\bTsunamiSecurityScanner\\b"},"operator":"match_regex"}],"transformers":[]},{"id":"ua0-600-54x","name":"Nimbostratus is a vulnerability scanner that scan websites unprompted","tags":{"type":"security_scanner","category":"attack_attempt"},"conditions":[{"parameters":{"inputs":[{"address":"server.request.headers.no_cookies","key_path":["user-agent"]}],"regex":"(?i)\\bnimbostratus-bot\\b"},"operator":"match_regex"}],"transformers":[]},{"id":"ua0-600-5xx","name":"Blind Sql Injection Brute Forcer","tags":{"type":"security_scanner","category":"attack_attempt"},"conditions":[{"parameters":{"inputs":[{"address":"server.request.headers.no_cookies","key_path":["user-agent"]}],"regex":"(?i)\\bbsqlbf\\b"},"operator":"match_regex"}],"transformers":[]},{"id":"ua0-600-6xx","name":"Suspicious user agent","tags":{"type":"security_scanner","category":"attack_attempt"},"conditions":[{"parameters":{"inputs":[{"address":"server.request.headers.no_cookies","key_path":["user-agent"]}],"regex":"mozilla/4\\.0 \\(compatible(; msie 6\\.0; win32)?\\)"},"operator":"match_regex"}],"transformers":[]},{"id":"ua0-600-7xx","name":"SQLmap","tags":{"type":"security_scanner","category":"attack_attempt"},"conditions":[{"parameters":{"inputs":[{"address":"server.request.headers.no_cookies","key_path":["user-agent"]}],"regex":"^sqlmap/"},"operator":"match_regex"}],"transformers":[]},{"id":"ua0-600-9xx","name":"Skipfish","tags":{"type":"security_scanner","category":"attack_attempt"},"conditions":[{"parameters":{"inputs":[{"address":"server.request.headers.no_cookies","key_path":["user-agent"]}],"regex":"(?i)mozilla/5\\.0 sf/"},"operator":"match_regex"}],"transformers":[]}]}
```

Comment:
It's a big json telling the native dependency what is considered as an attack.
In the next version this will be customizable by the user, and waaay later this will be controlled remotely, so stored in memory, not in disk.